data/data-pipeline/data_pipeline/etl/sources/national_risk_index/etl.py

```diff
@@ -27,54 +26,41 @@ class NationalRiskIndexETL(ExtractTransformLoad):
     AGRIVALUE_LOWER_BOUND = 408000
 
     def __init__(self):
+        # load YAML config
+        self.DATASET_CONFIG = super().yaml_config_load()
+
+        # define the full path for the input CSV file
         self.INPUT_CSV = self.get_tmp_path() / "NRI_Table_CensusTracts.csv"
 
+        # this is the main dataframe
+        self.df: pd.DataFrame
+
+        # Output score variables (values set on datasets.yml)
+        self.RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_FIELD_NAME: str
```

Comment:
Given how `yaml_config_load` works, I think these should be class variables and not instance variables --- it doesn't matter a ton, but no need to accidentally confuse ourselves later thinking they're instance vars when they're set at the class level.

Reply:
That's a good point.

Reply:
I'll admit I'm often guilty of setting instance variables because `self` is my default rather than `cls`.

Reply:
I would not hold up merging to change them --- just like, I was reading what `yaml_config_load` actually does and was like "one day some future person might, _might_ find this confusing"

Reply:
Great point, Changed.